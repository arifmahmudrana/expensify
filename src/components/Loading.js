import React from 'react';

const Loading = () => (
  <div className="loader">
    <img className="loader__img" src="/images/loader.gif" alt=""/>
  </div>
);

export default Loading;