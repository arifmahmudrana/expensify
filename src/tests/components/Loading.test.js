import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../components/Loading';

test('Loading should render correctly', () => {
  const wrapper = shallow(<Loading />);

  expect(wrapper).toMatchSnapshot();
});