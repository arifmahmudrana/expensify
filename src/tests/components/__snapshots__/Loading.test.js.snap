// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly 1`] = `
<div
  className="loader"
>
  <img
    alt=""
    className="loader__img"
    src="/images/loader.gif"
  />
</div>
`;
